### PR TITLE
feat: PrevNext Bottom Navigation

### LINKED ISSUE #92: Button style variation
A lot of the content pages have a "Previous" and a "Next" button at the bottom of the page.
They are simple links, with some additional styling:

https://www.mammotome.com/patient-resources/risk-factors/
Screenshot 2023-05-02 at 11 23 47

### PR DESCRIPTION
**Pull request by Acidham**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #92 

Test URLs:
- Before: https://main--mammotome--hlxsites.hlx.page/en/patient-resources/
- After: https://92-button-style-variation--mammotome--hlxsites.hlx.page/en/patient-resources/

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -78,13 +85,82 @@ function buildHeroBlock(main) {
   }
 }
 
+/**
+ * Build Prev Next Bottom Navigation
+ * @param main
+ * @returns {Promise<void>}
+ */
+async function buildPrevNext(main) {
```
Comment (solaris007) [inline scripts/scripts.js:93]: you could reduce redundancy and increase readability by using functional programming, for example in the area of creating the arrow and insertion of nav/arrow elements for prev and next box, plus using a guard clause:

```
async function buildPrevNext(main) {
  const moveHeaderLinkDiv = (el) => {
    const text = el.innerText;
    const newDiv = document.createElement('div');
    newDiv.classList.add('prev-next-header-link');
    newDiv.innerText = text;
    el.innerText = '';
    el.insertAdjacentElement('afterbegin', newDiv);
  };

  const buildNavButtons = async (key, defaultText) => {
    const newDiv = document.createElement('div');
    newDiv.classList.add('prev-next-button');
    newDiv.innerText = await translate(key, defaultText);
    return newDiv;
  };

  const createArrow = (direction) => {
    const arrow = document.createElement('div');
    arrow.classList.add(`${direction}-arrow`);
    arrow.innerHTML = direction === 'left' ? HTML_ARROW_PREV : HTML_ARROW_NEXT;
    return arrow;
  };

  const prevNext = main.querySelector('.columns.prev-next');
  if (!prevNext) return;

  const prevNextContainer = prevNext.firstElementChild;
  const verticalSeparator = document.createElement('div');
  verticalSeparator.classList.add('vertical-line');
  prevNextContainer.firstElementChild.insertAdjacentElement('afterend', verticalSeparator);

  const prevBox = prevNextContainer.firstElementChild.querySelector('a');
  const nextBox = prevNextContainer.lastElementChild.querySelector('a');

  if (prevBox) {
    moveHeaderLinkDiv(prevBox);
    prevBox.insertAdjacentElement('afterbegin', await buildNavButtons('navPreviousText', 'Previous'));
    prevBox.insertAdjacentElement('afterbegin', createArrow('left'));
  }

  if (nextBox) {
    moveHeaderLinkDiv(nextBox);
    nextBox.insertAdjacentElement('afterbegin', await buildNavButtons('navNextText', 'Next'));
    nextBox.insertAdjacentElement('afterbegin', createArrow('right'));
  }
}

```

File: styles/styles.css [hunk 2]
```diff
@@ -645,3 +658,106 @@ main .schedule-demo .default-content-wrapper h2 {
...
+.columns.prev-next {
+  display: flex;
+  justify-content: center;
+}
+
+/* Bottom Nav box */
+.columns.prev-next > div {
+  display: grid;
+  grid-template-columns: 1fr auto 1fr;
+  grid-auto-rows: auto;
```
Comment (solaris007) [inline styles/styles.css:672]: can be omitted as it is default

File: styles/styles.css [hunk 3]
```diff
@@ -645,3 +658,106 @@ main .schedule-demo .default-content-wrapper h2 {
...
+}
+
+@media screen and (min-width: 767px) {
+  .columns.prev-next > div {
+    width: 918px;
+  }
+}
+
+  /* Previous button box */
+.columns.prev-next > div > div:nth-child(1) > a {
```
Comment (solaris007) [inline styles/styles.css:685]: from here to the bottom some classes can be merged and the CSS compressed:

```
.columns.prev-next a .prev-next-button {
  color: var(--mt-prev-next-button-text-color);
  font-size: var(--mt-prev-next-button-font-size);
  font-weight: var(--mt-prev-next-button-font-weight);
  line-height: var(--mt-prev-next-button-line-height);
  letter-spacing: 1.7px;
  text-transform: uppercase;
  grid-row: 1 / 2;
}

.columns.prev-next a .prev-next-header-link {
  display: inline-block;
  overflow: hidden;
  text-overflow: ellipsis;
  direction: ltr;
  width: 100%;
  color: var(--mt-prev-next-header-link-text-color);
  line-height: var(--mt-prev-next-header-link-line-height);
  font-size: var(--mt-heading-font-size-m);
  font-weight: var(--mt-prev-next-header-link);
  white-space: nowrap;
  grid-row: 2 / 3;
}

.columns.prev-next .vertical-line {
  border-left: 1px solid var(--mt-global-color-line-grey);
  height: 100%;
  content: '';
  align-items: center;
}

.columns.prev-next > div > div:first-child > a,
.columns.prev-next > div > div:last-child > a {
  display: grid;
  grid-template-rows: auto auto;
  align-items: center;
  grid-column-gap: 5px;
  overflow: hidden;
}

.columns.prev-next > div > div:first-child > a {
  grid-template-columns: auto 1fr;
  .prev-next-header-link {
    text-align: left;
    padding-right: 5px;
  }
}

.columns.prev-next > div > div:last-child > a {
  grid-template-columns: 1fr auto;
  justify-items: end;
  .prev-next-header-link {
    text-align: right;
    padding-left: 5px;
  }
}

.columns.prev-next a .left-arrow,
.columns.prev-next a .right-arrow {
  grid-row: 1 / 3;
  align-items: center;
}

.columns.prev-next a .right-arrow {
  grid-column: 2 / 3;
}
```

File: styles/styles.css [hunk 4]
```diff
@@ -34,6 +34,9 @@
   --mt-global-color-silver: #ccc;
   --mt-global-color-alabaste: #F7F7F7;
   --mt-global-color-gallery: #ECECEC;
+  --mt-global-color-accent-grey: #BFBFBF;
+  --mt-global-color-accent-black: #3D3E40;
```
Comment (dzehnder) [inline styles/styles.css:38]: Same as `--mt-global-color-base-secondary`

**Comment by dzehnder** [comment]
Icons don't look exactly the same. Is that something the customer can live with?

Their page:
Screenshot 2023-05-08 at 14 49 17

Our page:
Screenshot 2023-05-08 at 14 49 27

**Review by dzehnder — DISMISSED** [review]

**Review by solaris007 — DISMISSED** [review]

**Review by dzehnder — APPROVED** [review]

**Pull request merged**